base/initdefs.jl

```diff
@@ -19,14 +19,17 @@ An array of the command line arguments passed to Julia, as strings.
 const ARGS = String[]
 
 """
-    exit(code=0)
+    exit(code=0; kill_tasks=true)
```

Comment:
It doesn't kill the tasks, just waits for them?

Reply:
The default is to kill all tasks (as before). The new optional behavior (kill_tasks=false) is to run the rest of the program to completion and only kill this task.